source/reference/method/MongoDBClient__construct.txt

```diff
@@ -124,6 +124,11 @@ Parameters
                 'root' => 'MongoDB\Model\BSONDocument',
             ]
 
+     * - builderEncoder
+       - MongoDB\\Builder\\Encoder
```

Comment:
The correct type is `MongoDB\Codec\Encoder`.
Specified here: https://github.com/mongodb/mongo-php-library/blob/dacd06e775354c5a79f8ee04746ad4f969aed70b/src/Client.php#L82

Reply:
Oh, I'll fix that.

Reply:
Fixed: https://github.com/mongodb/mongo-php-library/pull/1593